docs/articles.bib

```diff
@@ -90,6 +90,19 @@ @article{Breuer_2006_Optimal
 
 
 #Last name begins with C
+@article{Cavalcanti_2015_Detection,
```

Comment:
@Roshan-Thomas Thank you for this PR! 

Could you please make sure the keys are arranged alphabetically? You have placed them in the right spot but a key beginning with `cav` should be placed after `cab` and so on.

Reply:
Hi, I have fixed the issue in the following commit. Hope that fixes everything :)